compact/tof_endcap.xml

```diff
@@ -95,12 +110,13 @@
       <detector
         id="BackwardTOF_ID"
         name="BackwardTOF"
-        type="epic_TrapEndcapTracker"
+        type="epic_TOFEndcap"
         readout="TOFEndcapHits"
         vis="TOFVis"
         reflect="true">
         <module name="BackwardModule" vis="TOFModuleVis">
           <trd x1="BackwardTOFMod_x1/2.0" x2="BackwardTOFMod_x2/2.0" z="BackwardTOFMod_y/2"/>
+          <diskdimension rMin="BackwardTOF_rMin" rMax="BackwardTOF_rMax" xOffset="BackwardTOF_xOffset" det_height="BackwardTOF_det_height" zPos="BackwardTOF_zmin" cooling_plate_height="TOFEndcap_cooling_plate_height" wallthickness_coolingtube="TOFEndcap_wallthickness_coolingtube" diameter_coolingtube="TOFEndcap_diameter_coolingtube" material="Aluminum"/>
```

Comment:
Do we need to call this `diskdimension` instead of `dimension`? Most dd4hep geometry plugins use simply `dimension` so I'd prefer to stay with that approach.

Reply:
Done.